package.json

```diff
@@ -81,6 +81,7 @@
     "@lingui/react": "^4.11.2",
     "@redux-devtools/extension": "^3.2.2",
     "@reduxjs/toolkit": "^1.8.0",
+    "@storybook/addon-a11y": "^8.2.9",
```

Comment:
Why do we need this in dependencies ? It should go to dev dependency ?

Reply:
Right, Thank you for pointing that out! I've moved it to devDependencies as suggested.